### ISSUE TITLE
bug in config set --key url

### ISSUE THREAD

**Issue by sylvainfaivre** [pglombardo/pwpush-cli#655]
## 🐛 Bug Report

The command to set the url reports success but does nothing.

So I had to manually edit the config file in order to be able to use a self hosted instance.

## 🔬 How To Reproduce

```

$ bin/pwpush config set --key url --value https://example.com
Success

$ grep url ~/.config/pwpush/config.ini
url = False

$ bin/pwpush config show

=== Instance Settings:
Specify your credentials and even your private Password Pusher instance here.

┏━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃ Key   ┃ Value   ┃ Description                                                            ┃
┡━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
│ URL   │ False   │ The Password Pusher instance to work with.                             │
│ email │ Not Set │ E-mail address of your account on Password Pusher.                     │
│ token │ Not Set │ API token from your account.  e.g. 'https://pwpush.com/en/users/token' │
└───────┴─────────┴────────────────────────────────────────────────────────────────────────┘
<snip>

```

### Environment

* OS: Ubuntu 23.04
* Python 3.11.4

**Comment by sylvainfaivre** [comment]
ahem, maybe this bot is too sensitive

**Comment by pglombardo** [comment]
Agree - I don't like the bot wording & tone either :-/

Thanks for reporting @sylvainfaivre.  I just tried and got the same.  This CLI needs some updates and polishing.  I'll see if I can get some time to do this soon hopefully.

**Comment by pglombardo** [comment]
Updated the bot [message](https://github.com/pglombardo/pwpush-cli/blob/master/.github/workflows/greetings.yml#L14).

**Comment by pglombardo** [comment]
Fixed by #752 - thanks all for pointing this out (and the fix)!

**Comment by pglombardo** [comment]
Python package 0.10.0 with this fix is out: https://pypi.org/project/pwpush/
